src/components/LoginForm/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './index.css';
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { login: '', password: '' };
+  }
+
+  render() {
+    const onLoginChange = event => this.setState({ login: event.target.value });
```

Comment:
move it not to render method.
```suggestion
    class MyComponent extends React.Component {
    constructor() {}
    method = () => {}
    render() {}
}
```